internal/backend/db/dbgorm/triggers.go

```diff
@@ -125,29 +124,17 @@ func (db *gormdb) DeleteTrigger(ctx context.Context, id sdktypes.TriggerID) erro
 }
 
 func (db *gormdb) UpdateTrigger(ctx context.Context, trigger sdktypes.Trigger) error {
-	curr, err := db.GetTriggerByID(ctx, trigger.ID())
-	if err != nil {
-		return err
+	r, err := db.getTriggerByID(ctx, trigger.ID().UUIDValue())
+	if r == nil || err != nil {
+		return translateError(err)
 	}
 
-	if envID := trigger.EnvID(); envID.IsValid() && curr.EnvID() != envID {
-		return sdkerrors.ErrConflict
-	}
+	r.CodeLocation = trigger.CodeLocation().CanonicalString()
+	r.EventType = trigger.EventType()
+	r.Filter = trigger.Filter()
+	r.Schedule = trigger.Schedule()
```

Comment:
Your forgot `Name`, `UniqueName`, and `WebhookSlug`

Reply:
WebhookSlug cannot be updated.
I'll fix Name (and hence UniqueName).

Reply:
done